RetailCoder.VBE/Settings/GeneralConfigProvider.cs

```diff
@@ -53,20 +53,21 @@ private void CheckForEventsToRaise(GeneralSettings other)
         protected virtual void OnLanguageChanged(EventArgs e)
         {
             var handler = LanguageChanged;
-            if (handler != null)
-            {
-                handler(this, e);
-            }
+            LanguageChanged?.Invoke(this, e);
```

Comment:
@Vogel612 it is. https://www.google.ca/amp/s/codeblog.jonskeet.uk/2015/01/30/clean-event-handlers-invocation-with-c-6/amp/ the compiler invokes the handler only once.